src/components/common/modal/Modal.tsx

```diff
@@ -122,7 +123,7 @@ export const Modal = ({ children, isOpen, onClose, maxWidth }: ModalProps) => {
                 ref={modalRef}
                 role="button"
                 aria-labelledby={titleId}
-                className="modal-container"
+                className={`modal-container ${fullScreen ? 'fullscreen' : ''}`}
```

Comment:
use cn for this case, also maybe rename `fullScreen` variable to `fullscreen`? 
```suggestion
                className={cn('modal-container', {
                    fullscreen: fullScreen,
                })}
```